src/layout/SigneeList/SigneeListComponent.tsx

```diff
@@ -30,7 +30,12 @@ export function SigneeListComponent({ node }: PropsFromGenericComponent<'SigneeL
   });
 
   if (taskType !== ProcessTaskType.Signing) {
-    return <Lang id='signee_list.wrong_task_error' />;
+    return (
+      <Lang
+        id='signing.wrong_task_error'
+        params={['SigneeList']}
+      />
+    );
```

Comment:
This should probably be part of component validation, instead of just printed out to the user. Logged in DevTools, red border, all the bells and whistles. If we decide to integrate that log into (for example) application insights in the future, this thing won't circumvent that.